atmos_held_suarez_obs/create_artifact.jl

```diff
@@ -0,0 +1,33 @@
+using Downloads
+import Tar
+using ClimaArtifactsHelper
+
+const FILE_URL = "https://caltech.box.com/shared/static/qzmqmiskfrvwlzbbry1i4xfw1foqei4m.tar"
```

Comment:
Ideally, it would preferable if this script reproduced the data instead of downloading it.

Reply:
I agree, but reproducing the data takes a few hours so I don't think it is suitable for an artifact.

I have an [sbatch script](https://github.com/CliMA/ClimaAtmos.jl/pull/2895/files#diff-ed92f5beba79b5a3194353e3c4d3eea038586ee0d6fa2b9c8d35bec166a0720b) to reproduce the data, it could live here instead.

Reply:
It's fine if it takes a few hours. While this step will probably never be repeated, one should be able to do so. The main goal of ClimaArtifacts is make our input data files reproducible. Just adding a link to a file uploaded to Box does not help with explaning how the data was produced and does not enable others to reproduce it. 

You are welcome to instruct users on how to obtain/generate the data, as we do in the soil artifact: https://github.com/CliMA/ClimaArtifacts/blob/main/soil_params_Gupta2020_2022/create_artifacts.jl